README.md

```
@@ -1,18 +1,15 @@
 Majestik - a Magik implementation
 =================================
 
-Majestik is an attempt to implement the Magik programming language[^1], using ANTLR4[^2] and Java's Class-File API[^3].
+Majestik is an attempt to implement the Magik programming language[^1], using Magik-Tools[^2] and Java's Class-File API[^3].
 It is currently in the very early stages of development and lacks most features.
 
 
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)